src/components/TestToolMenu.js

```diff
@@ -43,7 +44,7 @@ const TestToolMenu = props => (
 
         {/* Option to switch between staging and default api endpoints.
         This enables QA and internal testers to take advantage of sandbox environments for 3rd party services like Plaid and Onfido. */}
-        <TestToolRow title="Use Staging Server">
+        <TestToolRow title="Use Staging Server" isHidden={CONFIG.IS_USING_LOCAL_WEB}>
```

Comment:
I don't think we should add an `isHidden` prop, but inline the expression: https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#inline-ternaries

There are a lot of examples where we render the loading indicator like this:
```jsx
<>
  {shouldShowLoadingIndicator && (
    <FullscreenLoadingIndicator
      style={[styles.opacity1, styles.bgTransparent]}
     />
  )}
</>
```